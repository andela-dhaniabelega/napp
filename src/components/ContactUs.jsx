import React from 'react';
import '../../public/sass/styles.scss';
import Header from '../components/Header';
import ContactUsSlider from '../components/ContactUsSlider';
import Footer from '../components/Footer';


/**
 * Stateless component
 * Simply renders contact information
 *
 * @return {void}
 */
const ContactUs = () => (
  <div className="wrapper">
    <div className="center">
      <Header />
      <ContactUsSlider />
      <div className="about-text">
        <p>
          You can reach us at:
        </p>
        <p>
          Phone: +2348146350497, +234883344585,<br />
          Email: info@thenappjournal.com <br />
          Address: 215, Ken Adams Street, Ikoyi Lagos.

        </p>
      </div>
      <Footer />
    </div>
  </div>
);

export default ContactUs;



