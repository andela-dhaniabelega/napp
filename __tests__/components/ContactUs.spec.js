import React from 'react';
import renderer from 'react-test-renderer'
import ContactUs from '../../src/components/ContactUs';


describe('Contact Us Page', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<ContactUs />).toJSON();
          expect(tree).toMatchSnapshot();
		});
});