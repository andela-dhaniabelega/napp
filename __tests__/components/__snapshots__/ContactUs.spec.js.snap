// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Us Page Contact Us renders a snapshot 1`] = `
<div
  className="wrapper"
>
  <div
    className="center"
  >
    <div
      className="header"
    >
      <div
        className="logo floatleft"
      >
        <a
          href="/"
        >
          <img
            alt=""
            src="../public/images/logo copy.png"
          />
        </a>
      </div>
      <div
        className="top_menu floatleft"
      >
        <ul>
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/about"
            >
              About Napp
            </a>
          </li>
          <li>
            <a
              href="/contact"
            >
              Contact us
            </a>
          </li>
          <li>
            <a
              onClick={[Function]}
              role="button"
            >
              Sign In with Google
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="slider-margin"
    >
      <div
        className="main_menu_area"
      >
        <ul
          id="nav"
        />
      </div>
      <div
        className="slider_area"
      >
        <div
          className="slider"
        >
          <ul
            className="bxslider"
          >
            <li>
              <img
                alt=""
                src="../public/images/contact.jpg"
              />
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="about-text"
    >
      <p>
        You can reach us at:
      </p>
      <p>
        Phone: +2348146350497, +234883344585,
        <br />
        Email: info@thenappjournal.com 
        <br />
        Address: 215, Ken Adams Street, Ikoyi Lagos.
      </p>
    </div>
    <div
      className="footer_bottom_area"
    >
      <div
        className="copyright_text"
      >
        <p>
          Copyright Â© 2017 Napp. All rights reserved
        </p>
      </div>
    </div>
  </div>
</div>
`;
